### ISSUE TITLE
Unsupported locale setting

### ISSUE THREAD

**Issue by danielhers** [maxzheng/memory-tools#2]
When running `show-mem` I get the following error:
```
Traceback (most recent call last):
  File ".virtualenv/bin/show-mem", line 7, in <module>
    from memorytools.show_mem import main
  File ".virtualenv/lib/python3.5/site-packages/memorytools/__init__.py", line 9, in <module>
    locale.setlocale(locale.LC_ALL, 'en_US')
  File ".virtualenv/lib/python3.5/locale.py", line 594, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
```
Indeed, I don't see `en_US` when running `locale -a`. However, I do see `en_US.utf8`. Perhaps this should be used instead if `en_US` doesn't exist?

**Comment by maxzheng** [comment]
memory-tools is used for Python 2.6 only as indicated in the README.  For newer versions, you can use newer and better tools: https://stackoverflow.com/questions/1435415/python-memory-leaks

**Comment by danielhers** [comment]
I did manage to run it successfully on Python 3.5, though, once I changed the `en_US` to `en_US.utf8`. Thanks for the link, though. It's very useful!

**Comment by maxzheng** [comment]
Looks like I did add support for Python 3.x awhile ago, but I don't have that problem when using Python 3.5. Just published 1.0.4 that sets to default locale instead of en_US. Let me know if that fixes it for u too.

**Comment by danielhers** [comment]
I upgraded to 1.0.4, but now it doesn't work at all:
```
Traceback (most recent call last):
  File ".virtualenv/bin/show-mem", line 7, in <module>
    from memorytools.show_mem import main
ImportError: No module named 'memorytools'
```
and indeed there is no directory `.virtualenv/lib/python3.5/site-packages/memorytools`, just `.virtualenv/lib/python3.5/site-packages/memory_tools-1.0.4.dist-info`.

**Comment by maxzheng** [comment]
That's really weird as the sources are in there. It's fixed in 1.0.5. Apparently, it gets fixed by the following change:

-  packages=setuptools.find_packages(''),
+  packages=setuptools.find_packages(),

**Comment by maxzheng** [comment]
Ic. When I moved the sources out of 'src', I changed find_packages('src') to just '', so it couldn't find the packages anymore as it should be the path to find packages. Oddly, it works locally and so no problems with unit tests.
